site/components/pages/Infinity/styles.ts

```diff
@@ -833,10 +824,60 @@ export const ctaSection = css`
   }
 `;
 
-export const footerWrapper = css`
-  grid-column: full;
-  grid-template-columns: inherit;
-  footer {
-    background-color: black;
+export const footerContainer = css`
+  padding-bottom: 3rem;
+  padding-top: 5rem;
```

Comment:
We try to use 8px grid for everything, so I think you should be able to do
`4.8rem` instead of 5 and `2.4rem` instead of 3

Reply:
Sure thing, I've updated to use rem values at '.4' intervals